src/Apps/Order2/Routes/Checkout/Components/Order2CheckoutLoadingSkeleton.tsx

```diff
@@ -88,29 +89,27 @@ const Order2CollapsibleOrderSummarySkeleton: React.FC<
       </Column>
 
       <Column
-        start={[1, 1, 8]}
         span={[12, 12, 5]}
```

Comment:
This one is the same question about leaving a column margins on the right as it looks in design. Maybe what we have here is good enough approximation here as well.